from pyzeebe.errors.pyzeebe_errors import PyZeebeError


class EnvironmentConfigError(PyZeebeError):
    pass
